docker-compose-all-in-one.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
```

Comment:
Is there a difference between this file and docker-compose.yml?  If there is a change, can we just change that file?

Reply:
No difference, I just want to be explict about which docker-compose.yml is being used by which deploy job. When this is merged, I can be more explicit in the deployment job's scipt, and then I'd like to deprecate/remove the vanilla docker-compose.yml. I'm leaving it in for now so the deploy-all-in-one job works until this is merged. One step at a time.